### PR TITLE
add support for resource creation

### PR DESCRIPTION
**Pull request by jelhan**

### CODE REVIEW

File: lib/responses/json.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+ *
+ * @param  {Object} data
+ */
+
+/**
+ * res.json() changed order of arguments a few times.
+ * Sails.js /lib/router/res.js provides a private function to normalize the arguments.
+ * https://github.com/balderdashy/sails/blob/0506f0681590dc92986985bc39609c88b718a997/lib/router/res.js#L329-L360
+ */
+/* eslint-disable */
```
Comment (IanVS) [inline lib/responses/json.js]: Just curious, why do you need to disable eslint for this section?

**Comment by IanVS** [comment]
You're on a role, this looks awesome.  👍

File: lib/responses/json.js (hunk 1 above)
Comment (jelhan) [inline lib/responses/json.js]: Don't want to change the function from Sails.js to keep it maintainable and there are some eslint issues in Sails.js code. E.q. the else will fail due to [1tbs rule](http://eslint.org/docs/rules/brace-style#tbs).

Comment (IanVS) [inline lib/responses/json.js]: Ah, makes perfect sense.

**Pull request merged**